src/Application/Error.jsx

```diff
@@ -27,30 +30,27 @@ const ErrorWrapper = styled.div`
     animation: ${slideIn} 500ms ease-in-out 1;
 
     width: ${props => {
-        if (props.width) return props.width
-        if (props.full) return "100%"
-        return "auto"
-    }};
+		if (props.width) return props.width
+		if (props.full) return "100%"
+		return "auto"
+	}};
 
     max-width: 250px;
+
+    z-index: 1000;
```

Comment:
z-index: 1000 <-- ich bin kein Fan von ich setzte einfach ein großen Wert und wird schon passen. Lukas hatte auch ganz viele 9999, und 999, und 99999. Das fällt ein später wieder auf die Füße. Beim z-index sollte es eine klare Ebenen Einteilung geben, die dann jeden bekannt ist und mit den Designern abgestimmt wurde.

Reply:
Können wir gerne machen, wo?

Reply:
Vorschlag: Suche mal nach z-index die habe ich schon mal sortiert gehabt und passe es auf eins der sinnvollen level an.

Ich erstelle ein Ticket, das wir eine Datei anlegen die z-index als ConstNames exportiert. Das wir dann z.B. an der Stelle z-index: LayerError zu stehen haben und LayerError ist dann irgendein kleiner Wert. 
Aber das wir insgesammt uns stark in den Ebenen reduzieren und den Rest durch css richtig verschieben und schachteln auflösen.

Reply:
13713